scripts/bundleSize/bundleSize.test.js

```diff
@@ -94,42 +113,38 @@ describe('bundleSize', () => {
   describe('when all service bundles are within the defined limits', () => {
     setUpFSMocks(145000, 150000);
 
-    it('should not throw an error', () => {
+    it('should not throw an error', async () => {
       let didThrow = false;
-      jest.isolateModules(() => {
-        try {
-          require('.'); // eslint-disable-line global-require
-        } catch (e) {
-          didThrow = true;
-        }
-      });
+      try {
+        const { default: bundleSize } = await import('./index.js');
+        bundleSize();
```

Comment:
for the mocks configured above to be used in the code we are tested we have to dynamically import it